tests/subyt/test_generator.py

```
@@ -66,76 +62,65 @@ def get_indicator_from_name(
-            ["data_glob/*.json"]
-        )  # insert "glob pattern" glob source
-        for inp_name in inp_names:
-            key = get_indicator_from_name(inp_name, fallback="_")
-            assert key not in inputs, (
-                f"duplicate key '{key}' for input '"
-                f"{inp_name}' --> object[{inputs[key]}]"
-            )
-            inputs[key] = SourceFactory.make_source(
-                os.path.join(inp_path, inp_name)
-            )
+def test_templates():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_templates`

Add return type annotation: `None`

(ANN201)